src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,215 @@
+        linkType: "internal",
+      },
+      {
+        label: "Termini e Condizioni",
+        href: "https://www.pagopa.it/it/termini-e-condizioni-di-utilizzo-del-sito/",
+        ariaLabel: "Vai al link: Termini e Condizioni",
+        linkType: "internal",
+      },
+      {
+        label: "Società trasparente",
+        href: "https://pagopa.portaleamministrazionetrasparente.it/pagina746_altri-contenuti.html",
+        ariaLabel: "Vai al link: Società trasparente",
+        linkType: "internal",
+      },
+      {
+        label: "Responsible Disclosure Policy",
+        href: "https://www.pagopa.it/it/responsible-disclosure-policy/",
+        ariaLabel: "Vai al link: Responsible Disclosure Policy",
+        linkType: "internal",
+      },
+      {
+        label: "Modello 321",
+        href: "https://pagopa.portaleamministrazionetrasparente.it/pagina746_altri-contenuti.htmls",
+        ariaLabel: "Vai al link: Modello 321",
+        linkType: "internal",
+      },
+    ],
+  },
+  // Fourth column
+  followUs: {
+    title: "Seguici su",
+    socialLinks: [
+      {
+        icon: "linkedin",
+        title: "LinkedIn",
+        href: "https://www.linkedin.com/company/pagopa/",
+        ariaLabel: "Link: vai al sito LinkedIn di PagoPA S.p.A.",
+      },
+      {
+        title: "Twitter",
+        icon: "twitter",
+        href: "https://twitter.com/pagopa",
+        ariaLabel: "Link: vai al sito Twitter di PagoPA S.p.A.",
+      },
+      {
+        icon: "instagram",
+        title: "Instagram",
+        href: "https://www.instagram.com/pagopa/",
+        ariaLabel: "Link: vai al sito Instagram di PagoPA S.p.A.",
+      },
+      {
+        icon: "medium",
+        title: "Medium",
+        href: "https://medium.com/pagopa",
+        ariaLabel: "Link: vai al sito Medium di PagoPA S.p.A.",
+      },
+    ],
+    links: [
+      {
+        label: "Accessibilità",
+        href: "https://form.agid.gov.it/view/eca3487c-f3cb-40be-a590-212eafc70058/",
+        ariaLabel: "Vai al link: Accessibilità",
+        linkType: "internal",
+      },
+    ],
+  },
+};
+
+const AppFooter = () => {
+  return (
+    <Footer
+      loggedUser={false}
+      languages={lngs}
+      onLanguageChanged={() => {}}
+      companyLink={pagoPALink}
+      postLoginLinks={postLoginLinks}
+      preLoginLinks={preLoginLinks}
+      currentLangCode="it"
+      onExit={(exitAction) => {
+        console.log("Executing exit Action");
+        exitAction();
+      }}
+      productsJsonUrl="https://dev.selfcare.pagopa.it/assets/products.json"
+      hideProductsColumn={false}
+      legalInfo={
+        <span>
+          <strong>PagoPA S.p.A.</strong> - Società per azioni con socio unico -
+          Capitale sociale di euro 1,000,000 interamente versato - Sede legale
+          in Roma, Piazza Colonna 370, CAP 00187 - N. di iscrizione a Registro
+          Imprese di Roma, CF e P.IVA 15376371009
+        </span>
+      }
+    />
```

Comment:
mui-italia non avvolge direttamente il componente `Footer` nel tag html footer, consigliamo di farlo qui:

```suggestion
    <footer>
      <Footer
        loggedUser={false}
        languages={lngs}
        onLanguageChanged={() => {}}
        companyLink={pagoPALink}
        postLoginLinks={postLoginLinks}
        preLoginLinks={preLoginLinks}
        currentLangCode="it"
        onExit={(exitAction) => {
          console.log("Executing exit Action");
          exitAction();
        }}
        productsJsonUrl="https://dev.selfcare.pagopa.it/assets/products.json"
        hideProductsColumn={false}
        legalInfo={
          <span>
            <strong>PagoPA S.p.A.</strong> - Società per azioni con socio unico - Capitale sociale di euro 1,000,000
            interamente versato - Sede legale in Roma, Piazza Colonna 370, CAP 00187 - N. di iscrizione a Registro
            Imprese di Roma, CF e P.IVA 15376371009
          </span>
        }
      />
    </footer>
```